js/main.js

```diff
@@ -85,41 +85,39 @@ function DOMObjectLoaded(event) {
   }
 }
 
-var $newEntry = document.querySelector('.new-entry');
-var $noEntries = document.querySelector('.noEntries');
-
 function handleEntries(event) {
+  data.view = 'entries';
   if (data.entries.length === 0) {
     $noEntries.classList.remove('hidden');
   }
-  $entryForm.classList.toggle('hidden');
-  $entriesView.classList.remove('hidden');
-  $newEntry.classList.toggle('hidden');
-  $entriesNewButton.classList.remove('hidden');
-  $entries.classList.toggle('hidden');
-  data.view = 'entries';
+  for (var i = 0; i < $view.length; i++) {
+    setDataView($view[i]);
+  }
```

Comment:
You should do this loop inside of the `setDataView` function so that you can just call the function directly